modules/nw-ingress-controller-configuration-gcp-global-access.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-ingress-controller-configuration-gcp-global-access_{context}"]
+= Configuring global access for an Ingress Controller on GCP
+An Ingress Controller created on GCP with an internal load balancer generates an internal IP address for the service. A cluster administrator can specify the global access option, which enables clients in any region within the same VPC network and compute region as the load balancer, to reach the workloads running on your cluster.
+
+For more information, please reference the GCP documentation for link:https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access[Global access].
```

Comment:
* please reference -> see
* Global -> global if you're doing it in a running sentence as opposed to saying the title of the page

Reply:
So, I double checked the EP about Global Access v. global access, and it looks like they're capitalizing it when its in a running sentence. Should I double check this casing with someone? Google docs lowercases in a sentence and in the title. I think I'm going to use Global Access everywhere for now to be consistent.